packages/fe-fpm-writer/src/filter/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { CustomElement, InternalCustomElement, Position, EventHandler } from '../common/types';
+
+export interface CustomFilter extends CustomElement, EventHandler {
```

Comment:
Please add JSDoc comments for each property. It is not fully clear to what e.g. `ControlId` would refer to.

Reply:
JSDoc added.